infra/core/psn/spoke/prod/monitoring.tf

```diff
@@ -97,3 +97,22 @@ resource "azurerm_private_endpoint" "ampls" {
 
   tags = local.tags
 }
+
+# Container Apps Environment Log Analytics Workspace
+resource "azurerm_log_analytics_workspace" "cae_log_analytics" {
```

Comment:
We already have a log analytics: https://github.com/pagopa/io-wallet/blob/cc0a3a9cc617eb5038bede86cef33d6684fcf335/infra/core/psn/spoke/prod/monitoring.tf#L11